plugins/extractors/optimus/optimus.go

```diff
@@ -149,8 +149,9 @@ func (e *Extractor) buildJob(ctx context.Context, jobSpec *pb.JobSpecification,
 		Ownership: &facets.Ownership{
 			Owners: []*facets.Owner{
 				{
-					Urn:  jobSpec.Owner,
-					Name: jobSpec.Owner,
+					Urn:   jobSpec.Owner,
+					Name:  jobSpec.Owner,
```

Comment:
Since we already have Email field and optimus does not store name, should we just skip Name? Wdyt? @mabdh

Reply:
yes, fair enough.